ikarus/utils/eigendunetransformations.hh

```diff
@@ -58,4 +59,16 @@ namespace Ikarus {
     return {vec.data(), size};
   }
 
+  /** \brief Creates a Eigen::Vector from a given Dune::BlockVector. Each block should be a
+   * Dune::RealTuple of a given blockSize */
+  template <typename ScalarType, int blockSize>
+  Eigen::VectorX<ScalarType> convertDuneBlockToEigenVector(

```

Comment:
I think this implementation is not needed. I found the better founction which does no copy `viewAsFlatEigenVector.

Reply:
Yes, I know. I have removed it. I will commit it soon